src/data/ACTIONS/layers/patch7.05.ts

```diff
@@ -1,10 +1,244 @@
+		},
+		FLANKSTING_STRIKE: {
+			potencies: [
+				// untraited Dread Fangs is the "base GCD" here
+				{
+					value: 120,
+					bonusModifiers: [],
+				},
+				// with venom, the base gets +100p
+				{
+					value: 220,
+					bonusModifiers: [],
+					baseModifiers: ['FLANKSTUNG_VENOM'],
+				},
+				// the base potency of Hindsting Strike itself is treated as a "combo"
+				{
+					value: 340,
+					bonusModifiers: [BonusModifier.COMBO],
+					baseModifiers: [],
+				},
+				// +60p with positional
+				{
+					value: 400,
+					bonusModifiers: [BonusModifier.COMBO, BonusModifier.POSITIONAL],
+					baseModifiers: [],
+				},
+				// +100p with venom
+				{
+					value: 440,
+					bonusModifiers: [BonusModifier.COMBO],
+					baseModifiers: ['FLANKSTUNG_VENOM'],
+				},
+				// +160p with venom + positional
+				{
+					value: 500,
+					bonusModifiers: [BonusModifier.COMBO, BonusModifier.POSITIONAL],
+					baseModifiers: ['FLANKSTUNG_VENOM'],
+				},
+			],
+		},
+		HINDSBANE_FANG: {
+			potencies: [
+				// untraited Dread Fangs is the "base GCD" here
+				{
+					value: 120,
+					bonusModifiers: [],
+				},
+				// with venom, the base gets +100p
+				{
+					value: 220,
+					bonusModifiers: [],
+					baseModifiers: ['HINDSBANE_VENOM'],
+				},
+				// the base potency of the skill itself is treated as a "combo"
+				{
+					value: 340,
+					bonusModifiers: [BonusModifier.COMBO],
+					baseModifiers: [],
+				},
+				// +60p with positional
+				{
+					value: 400,
+					bonusModifiers: [BonusModifier.COMBO, BonusModifier.POSITIONAL],
+					baseModifiers: [],
+				},
+				// +100p with venom
+				{
+					value: 440,
+					bonusModifiers: [BonusModifier.COMBO],
+					baseModifiers: ['HINDSBANE_VENOM'],
+				},
+				// +160p with venom + positional
+				{
+					value: 500,
+					bonusModifiers: [BonusModifier.COMBO, BonusModifier.POSITIONAL],
+					baseModifiers: ['HINDSBANE_VENOM'],
+				},
+			],
+		},
+		FLANKSBANE_FANG: {
+			potencies: [
+				// untraited Dread Fangs is the "base GCD" here
+				{
+					value: 120,
+					bonusModifiers: [],
+				},
+				// with venom, the base gets +100p
+				{
+					value: 220,
+					bonusModifiers: [],
+					baseModifiers: ['FLANKSBANE_VENOM'],
+				},
+				// the base potency of skill itself is treated as a "combo"
+				{
+					value: 340,
+					bonusModifiers: [BonusModifier.COMBO],
+					baseModifiers: [],
+				},
+				// +60p with positional
+				{
+					value: 400,
+					bonusModifiers: [BonusModifier.COMBO, BonusModifier.POSITIONAL],
+					baseModifiers: [],
+				},
+				// +100p with venom
+				{
+					value: 440,
+					bonusModifiers: [BonusModifier.COMBO],
+					baseModifiers: ['FLANKSBANE_VENOM'],
+				},
+				// +160p with venom + positional
+				{
+					value: 500,
+					bonusModifiers: [BonusModifier.COMBO, BonusModifier.POSITIONAL],
+					baseModifiers: ['FLANKSBANE_VENOM'],
+				},
+			],
+		},
+
+		STEEL_MAW: {statusesApplied: ['HONED_REAVERS']},
+		DREAD_MAW: {id: SHARED.UNKNOWN.id},
```

Comment:
```suggestion
		DREAD_MAW: SHARED.UNKNOWN,
```

Reply:
That throws an error, looking at how Endwalker handled it, all we did was mark the id as unkown.

Reply:
or maybe not, despite it yelling at me that Shared.Unknown is unassignable to fields, it still compiles and doesn't appear to break anything

Reply:
The necessary change is to use the spread operator to set to SHARED.UNKNOWN and then unset any fields that are on the base object but aren't on the SHARED.UNKNOWN object.

So for these, should generally be `{...SHARED.UNKNOWN, onGcd: undefined, speedAttribute: undefined}`

Reply:
changes made, out of curiousity, why do we not just set all the fields of shared.Unknown to undefined?